src/components/Releases.js

```diff
@@ -0,0 +1,144 @@
+}
+.heading-secondary {
+    font-size: 3rem;
+    margin-bottom: 2rem;
+}
+.category {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+}
+.category:not(:last-child) {
+    border-bottom: 1px solid #ccc;
+}
+.frameworks {
+    padding: 3rem 0;
+}
+.icon {
+    display: inline-block;
+    width: 4rem;
+    height: 4rem;
+    background-size: contain;
+    background-repeat: no-repeat;
+}
+.glsp {
+    background-image: url(${GLSP});
+    vertical-align: bottom;
+}
+.emfcloud {
+    background-image: url(${EMFCLOUD});
+    vertical-align: bottom;
+}
+`
+
+const communityReleases = [
+    {
+        name: 'Theia 1.29.x (September 2022) - planned',
+        releasedate: 'September 29th, 2022',
+        releasecandidatedate:'August 25th, 2022',
+        frameworks: [
+            {
+                title: 'Eclipse GLSP',
+                url: 'https://www.eclipse.org/glsp/',
+                version: 'TBD',
+                icon: 'glsp'
+            },
+            {
+                title: 'Eclipse EMF.cloud',
+                url: 'https://www.eclipse.org/emfcloud/',
+                version: 'TBD',
+                icon: 'emfcloud'
+            }
+        ]
+    }
+]
+const Releases = () => (
+    <StyledResources>
+        <header className='header' role="banner">
+            <div className="row">
+                <Nav shouldRenderLogo={true} />
+                <div className="header__text-box">
+                    <h1 className="heading-primary">
+                        Releases
+                    </h1>
+                    <p>Eclipse Theia follows a monthly release cycle. You can find all releases including change logs and breaking changes <a href="https://github.com/eclipse-theia/theia/releases">here</a></p>
+                    <br></br>
+                    <p>In addition, Theia provides a "community release" every three month with extra focus on stability and compatibility. The list below shows all community releases including the next planned. In addition, you find technologies that integrate with Theia including the respective version that is supposed to be compatible with the community release. This compatibility information is provided and ensured by the respective project and therefore not in scope for the Theia project team. If you find issues with a technology, please report at the respective project. If you are interested in getting a technology listed, please open a PR to <a href="https://github.com/eclipse-theia/theia-website">this repo</a> to be listed.</p>
+
+                </div>
+            </div>
+        </header>
+        {communityReleases.map(({ name, frameworks, releasedate, releasecandidatedate }, i) => {
+            return (
+                <section key={i} className="category row" id="frameworks">
+                    <h2 className="heading-secondary">{name}</h2>
+                    <p>Release Date: {releasedate}</p>
+                    <p>Release Candidate Date: {releasecandidatedate}</p>
```

Comment:
minor: for newcomers do we want to mention this is the "community" release?

```suggestion
                    <p>Community Release Candidate Date: {releasecandidatedate}</p>
```